src/profile-logic/profile-data.js

```diff
@@ -2111,14 +2111,18 @@ export function getFriendlyThreadName(
   return label;
 }
 
-export function getThreadProcessDetails(thread: Thread): string {
-  let label = `thread: "${thread.name}"`;
+export function getThreadProcessDetails(
+  thread: Thread,
+  friendlyThreadName: string
+): string {
+  let label = `${friendlyThreadName}\n`;
```

Comment:
The only small nit would be that it would be nice to see the scheme for the eTLD+1 here (and actually in other places where getFriendlyThreadName is used, eg tooltips).

For that, you'd need to remove the scheme in `getGlobalTrackName` and `getLocalTrackName` (in `tracks.js`) instead of `getFriendlyThreadName`.

What do you think?